http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -102,17 +103,30 @@ class CollectorService[F[_]: Sync](
       )
       headerList = List(
         setCookie.map(_.toRaw1),
-        cacheControl(pixelExpected).map(_.toRaw1)
+        cacheControl(pixelExpected).map(_.toRaw1),
+        accessControlAllowOriginHeader(request).some,
+        `Access-Control-Allow-Credentials`().toRaw1.some
```

Comment:
Since we want to have drop-in replacement for akka-collector, I just copied the behavior exactly from akka-collector. This is how it is done in there [at the moment](https://github.com/snowplow/stream-collector/blob/master/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala#L155).